rules/codelingo/effective-go/good-package-name/codelingo.yaml

```diff
@@ -23,17 +23,27 @@ tenets:
       codelingo/docs:
         title: Good Package Name
         body: |
-          It's helpful if everyone using the package can use the same name 
-          to refer to its contents, which implies that the package name should 
-          be good: short, concise, evocative. By convention, packages are 
-          given lower case, single-word names; there should be no need for 
-          underscores or mixedCaps. Err on the side of brevity, since everyone 
-          using your package will be typing that name. And don't worry about 
+          A package name should be short and concise. Everyone using the
+          package should use the same name to refer to its contents.
+          By convention, packages are given lower case, single-word names. 
+          There is no need for underscores or mixedCaps. Do not worry about 
           collisions a priori. The package name is only the default name for 
           imports; it need not be unique across all source code, and in the 
           rare case of a collision the importing package can choose a different 
-          name to use locally. In any case, confusion is rare because the file 
-          name in the import determines just which package is being used.
+          name to use locally. In any case, confusion is rare because the file
+          name in the import determines which package is being used.
+
+          ## Bad code
+          ```golang
+          // list_to_implement_a_linked_list is a package that implements a doubly linked list.
+          package list_to_implement_a_linked_list
+          ```
+
+          ## Good code
+          ```golang
+          // list is a package that implements a doubly linked list.
```

Comment:
Then we can show that `linkedlist` smashed together as one word is the canonical substitute.
```
// Package linkedlist implements a doubly-linked list.
package linkedlist
```